src/sbmlutils/report/sbmlreport.py

```diff
@@ -73,6 +75,7 @@ def create_report(
     output_dir: Path,
     promote: bool = False,
     template: str = "report.html",
+    math_type: str = "cmathml",
```

Comment:
document in doc string and add a sanity check for the input parameter along the line of
```
math_types = ["cmathml", "pmathml", "latex"]
if math_type not in math_types:
    raise ValueError(f"math_type '{math_type}' not in supported types: '{math_types}')
```